features/fields.feature

```diff
@@ -314,5 +314,75 @@ Functionaliteit: Fields
       | reden.code         | O          |
       | reden.omschrijving | overlijden |
 
+  Rule: Wanneer een specifiek veld van een datum wordt gevraagd, dan worden velden die bij dat datumtype required zijn ook geleverd
+
+    Abstract Scenario: vragen om enkel(e) veld(en) van een datum
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde                |
+      | burgerservicenummer | <burgerservicenummer> |
+      En de persoon heeft de volgende 'geboorte' gegevens
+      | naam  | waarde      |
+      | datum | <GBA datum> |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | <burgerservicenummer>           |
+      | fields              | burgerservicenummer,<fields>    |

```

Comment:
De automation code is aangepast zodat het mogelijk is om de 'Dan' stappen als volgt te schrijven:

`Dan heeft de response een persoon met alleen de volgende 'geboorte' gegevens`

De eerste 'Dan' stap is dan niet meer nodig en burgerservicenummer kan uit de fields parameter

Reply:
dat is mooier. pas het hier en op andere plekken in de feature aan